### PR TITLE
Reworked Component Hierarchy

### PR DESCRIPTION
**Pull request by ryzokuken**

### CODE REVIEW

File: app/components/partials/Navbar.js [hunk 1]
```diff
@@ -1,21 +1,42 @@
...
-    <div className="spacer" />
-    <ul>
-      <li className="list-item"><a href="#"><span className="focus">8</span></a></li>
-      <li className="list-item"><a href="#">Preferences</a></li>
-      <li className="list-item"><a href="#"> {/* TODO: Remove Harcoded Value */} Logout (Ujjwal Sharma)</a></li>
-    </ul>
+    <div className="vertical-center logo text-center"><a href="#">Hermes</a></div>
+    <img onClick={toggleDropdown}
+         className="dropdown-trigger"
+         src="https://cdnjs.cloudflare.com/ajax/libs/material-design-icons/3.0.1/navigation/1x_web/ic_menu_white_48dp.png" />
```
Comment (526avijitgupta) [inline app/components/partials/Navbar.js]: How about downloading the icons and using those instead of a CDN link directly ?

**Review by 526avijitgupta — COMMENTED** [review]
Generally looks good code-wise. I'll try and run it once and leave more comments if there's something else.

File: app/components/partials/Navbar.js (hunk 1 above)
Comment (ryzokuken) [reply app/components/partials/Navbar.js]: yeah, that's what i had in mind. will make another commit tomorrow

**Comment by 526avijitgupta** [comment]
Just tried running it. Could you resize the menu and close icons to be the same as the font size used in the topbar ? The icons currently seem to be too large!

**Review by 526avijitgupta — APPROVED** [review]

**Pull request merged**